src/sec.c

```diff
@@ -23,7 +23,7 @@ sec_bin_init(sec_bin_t *bin) {
 bool
 sec_init(tsdn_t *tsdn, sec_t *sec, base_t *base, pai_t *fallback,
     const sec_opts_t *opts) {
-	assert(opts->max_alloc > 0);
+	assert(opts->max_alloc >= PAGE);
```

Comment:
I decided to make this requirement more stringent, because if `opts->max_alloc` is less than PAGE, there's something fishy going on. SEC can be disabled by setting number of shards to zero, otherwise `max_alloc` should have value >= PAGE.